src/main/java/webserver/ContentType.java

```diff
@@ -0,0 +1,44 @@
+package webserver;
+
+import java.util.Arrays;
+
+public enum ContentType {
+    JS("application/javascript", ".js", "/static"),
+    CSS("text/css;charset=utf-8", ".css", "/static"),
+    PNG("image/png", ".png", "/static"),
+    ICO("image/avif", ".ico", "/static"),
+    HTML("text/html;charset=utf-8", ".html", "/templates"),
+    FONT("font/woff", ".woff", "/static");
+
+    private String mimeType;
+    private String extension;
+    private String path;
+    private final String RESOURCES_PATH = "src/main/resources";
+
+    ContentType(String mimeType, String extension, String path) {
+        this.mimeType = mimeType;
+        this.extension = extension;
+        this.path = path;
+    }
+
+    public static ContentType of(String requestPath, String accept) {
```

Comment:
```suggestion
    public static ContentType of(String extention, String accept) {
```
URL의 확장자를 찾는 책임을 외부에 넘기면 코드가 더 깔끔해지지 않을까요? (아닐 수도 있음)

Reply:
이 메소드 아이디어와 구현 방식은 좋은 것 같아요.  👍